### PR TITLE
Ignore sound with wrong sample rate

### PR DESCRIPTION
**Pull request by ivan-ushakov**
1. At least on file `RESOURCE/Sound/EFF/dirigible1.wav` has sample rate 44100, let's ignore such files for now
2. Also increased number of channels to 32

### CODE REVIEW

**Comment by stalkerg** [comment]
I think it better to change the amount of channels where you call `SoundInit`

File: MechoSoma/XTool/xsound.cpp [hunk 1]
```diff
@@ -41,7 +41,11 @@ void *GetSound(int channel) { return soundManager ? soundManager->getSound(chann
 
 void SoundLoad(char *filename, void **lpDSB) {
   assert(soundManager != nullptr);
-  *lpDSB = soundManager->loadSound(filename);
+  try {
+    *lpDSB = soundManager->loadSound(filename);
```
Comment (caiiiycuk) [inline MechoSoma/XTool/xsound.cpp]: может сделаем что бы loadSound возвращал nullptr ? и избавимся от try/catch

**Comment by ivan-ushakov** [comment]
> I think it better to change the amount of channels where you call `SoundInit`

Idea was to keep original code approach, but I did this change anyway

File: MechoSoma/XTool/xsound.cpp (hunk 1 above)
Comment (ivan-ushakov) [reply MechoSoma/XTool/xsound.cpp]: Сделал

**Pull request merged**